src/comments/livetime.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import moment from 'moment'
+
+class LiveTime extends React.Component {
```

Comment:
Wasn't this a Utility component in CGC? It seems to make more sense in the Utilities namespace than in the Comments namespace.